ComponentKit/Core/CKComponentLifecycleManager.mm

```diff
@@ -62,6 +66,7 @@ - (instancetype)initWithComponentProvider:(Class<CKComponentProvider>)componentP
   if (self = [super init]) {
     _componentProvider = componentProvider;
     _sizeRangeProvider = sizeRangeProvider;
+    [CKComponentDebugController registerReflowListener:self];
```

Comment:
Though it's a bit of a shame to have `CKComponentLifecycleManager` aware of debug code I think it's totally worthwhile considering how much it simplifies `CKComponentDebugController`.